components/higher-order/with-filters/test/index.js

```diff
@@ -1,35 +1,62 @@
 /**
  * External dependencies
  */
-import { mount, shallow } from 'enzyme';
+import { shallow } from 'enzyme';
+import TestUtils from 'react-dom/test-utils';
+import ReactDOM from 'react-dom';
```

Comment:
I guess this shouldn't refer `react-dom` directly, but rather `@wordpress/element`. I'm not quite sure what is the best way to handle it.

Reply:
hmm sure, but then we'd be indirectly testing the export from `@wordpress/element` right?  This is being used more as a test utility in this case?

Reply:
It’s fine as is for the time being. We need to create our own lib anyways 😃